### ISSUE TITLE
Documentation template evaluation

### ISSUE THREAD

**Issue by zdxd** [jpmorganchase/salt-ds#1150]
Objective:
To evaluate the usability of component documentation and make recommendations that improve usability/task success.

```[tasklist]
### Part 2 (design variations): comparative usability test / comparing the pros and cons of different design options
- [x] Brief / methodology
- [x] Recruitment (in progress)
- [x] Interviews & Analysis (iterativley) Stop recruitment -> interview -> analysis cycle when data saturation occurs
- [x] Final pull together of results
- [x] Present / share results as needed
```

**Comment by zdxd** [comment]
on hold / awaiting prototypes to test

**Comment by zdxd** [comment]
same as above

**Comment by parsakhan21** [comment]
Feb 22 - can start template evaluation once green light received from Robin

**Comment by zdxd** [comment]
Shey & I discussed - will meet once template green lighted/delivered to plan out research

We will consider
- what can we do in parallel to design work?
- how will we evaluate design options (design options to be delivered mid-March)

**Comment by JohannaJoergensen** [comment]
24 Feb - Sion working on prototypes; should be available by next Mon/Tues.

**Comment by zdxd** [comment]
Shey & I meeting wed 1st to kick-off research

**Comment by zdxd** [comment]
Re-drafting brief to align with design work - will share with team before eow

**Comment by zdxd** [comment]
March 6th - met with Rufaro & Ben. They will send over any agreed updates to the template as and when they are approved for testing.  We will meet wk of 13th to update testing plan & revise timelines

**Comment by zdxd** [comment]
Progress  to recruitment paused due to content edits/updates

**Comment by zdxd** [comment]
Working on interview script, design and copy for testing to be delivered monday 20th. 

Zoe, Shey conducting pilot interview with Fernanda on Monday 20th. 

Post interview if required: Zoe to adjust scripts and Shey to adjust prototypes 

Recruitment to begin tomorrow ideally interviews begin on wed 22nd

**Comment by zdxd** [comment]
Interviews scheduled - still looking for participants so please share any contacts/recruitment sources

**Comment by zdxd** [comment]
@shey-v is this duplicating #1349   now?

**Comment by parsakhan21** [comment]
April 3- zoe working on insights, shey and zoe to meet tomorrow

**Comment by shey-v** [comment]
April 4 - meeting moved to Wednesday as Zoe needs a peer review. Zoe will share early findings and analysis with me tomorrow.

**Comment by shey-v** [comment]
April 5 - Met with Zoe to discuss key insights generated from the user testing sessions, we've now created some opportunity statements and Shey is ideating from there, to finalise the component page designs.

**Comment by shey-v** [comment]
April 6 - Shey & Zoe met with content team this morning to discuss findings from user testing. Shey updating designs. Will have this done by EOD Tuesday (considering the 4 day easter break).

**Comment by shey-v** [comment]
April 11 - Feedback received, component page designs (MVP1) finalised and socialised, Shey now creating a mobile view mockup.

**Comment by shey-v** [comment]
April 13 - Mobile versions have been mocked up, feedback received from Jo and Ben, Shey making updates.
